sdk/src/wallet/account/operations/output_consolidation.rs

```diff
@@ -70,14 +102,8 @@ where
     /// Consolidates basic outputs with only an [AddressUnlockCondition] from an account by sending them to an own
     /// address again if the output amount is >= the output_consolidation_threshold. When `force` is set to `true`, the
```

Comment:
Now only if no target_address is provided

Reply:
Changed